Helper/productData.js

```diff
@@ -3,17 +3,17 @@
 var data;
```

Comment:
this file will be removed eventually, it is just a placeholder until it is connected to database